### PR TITLE
Azure NSG Documenation missing some fields

### PR DESCRIPTION
**Pull request by ZhijunZhao**
##### SUMMARY

Fix bug #29586

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

azure_rm_securitygroup.py

##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = None
  configured module search path = [u'/Users/zhijzhao/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.14 (default, Sep 25 2017, 09:53:22) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)]
```

##### ADDITIONAL INFORMATION

I think documentation for `direction` and `access` is already pretty clear. 

```

```

### CODE REVIEW

**Comment by ZhijunZhao** [comment]
@yuwzho @zikalino @yaweiw

File: lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py [hunk 1]
```diff
@@ -87,11 +87,15 @@
                 default: "*"
             source_address_prefix:
                 description:
-                  - IP address or CIDR from which traffic originates.
+                  - The CIDR or source IP range. Asterix '*' can also be used to match all source IPs. Default tags such
+                    as 'VirtualNetwork', 'AzureLoadBalancer' and 'Internet' can also be used. If this is an ingress rule
+                    , specifies where network traffic originates from.
                 default: "*"
```
Comment (ZhijunZhao) [inline lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py]: The description is copied from Azure python SDK notes.

File: lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py [hunk 2]
```diff
@@ -87,11 +87,15 @@
...
             source_address_prefix:
                 description:
-                  - IP address or CIDR from which traffic originates.
+                  - The CIDR or source IP range. Asterix '*' can also be used to match all source IPs. Default tags such
```
Comment (jborean93) [inline lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py]: It is best to split each sentence up as a new list entry, e.g.

```
- The CIDR or source IP range.
- Asterix ...
- ...
```

When referring to values, enclosing them in `C()`, like `C(*)`, makes them highlighted in the actual docs so is a better choice than just quoting them. Not related to this but you can also use `I()` when referencing other options.

**Comment by ZhijunZhao** [comment]
ready_for_review

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
rebuild_merge

**Comment by jborean93** [comment]
looks like the bot isn't responding, will merge manually.

**Pull request merged**